logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -58,11 +57,11 @@ interface ConnectionRepository {
     suspend fun fetchSelfUserConnections(): Either<CoreFailure, Unit>
     suspend fun sendUserConnection(userId: UserId): Either<CoreFailure, Unit>
     suspend fun updateConnectionStatus(userId: UserId, connectionState: ConnectionState): Either<CoreFailure, Connection>
-    suspend fun getConnections(): Either<StorageFailure, Flow<List<ConversationDetails>>>
+    fun getConnections(): Either<StorageFailure, Flow<List<ConversationDetails>>>
     suspend fun insertConnectionFromEvent(event: Event.User.NewConnection): Either<CoreFailure, Unit>
-    suspend fun observeConnectionList(): Flow<List<ConversationDetails>>
```

Comment:
Why are you removing the suspend annotation? It's there for compatibility with JS  since all database operations are async there.